interpretation/deepseismic_interpretation/azureml_pipelines/README.md

```diff
@@ -0,0 +1,212 @@
+
+Please refer to microsoft docs for additional information on AzureML pipelines and related capabilities https://docs.microsoft.com/en-us/azure/machine-learning/concept-ml-pipelines
+
+## Files needed for this AzureML run
+
+You will need the following files to complete an run in AzureML
+
+- [.azureml/config.json](../../../.azureml.example/config.json) This is used to import your subscription, resource group, and AzureML workspace
+- [.env](../../../.env.example) This is used to import your environment variables including blob storage information and AzureML compute cluster specs
+- [kickoff_train_pipeline.py](dev/kickoff_train_pipeline.py) This script shows how to run an AzureML train pipeline 
+- [cancel_run.py](dev/cancel_run.py) This script is used to cancel an AzureML train pipeline run
+- [base_pipeline.py](base_pipeline.py) This script is used as a base class and kickoff_train_pipeline.py inherits from it. This is intended to be a helpful abstraction that an an future addition of an inference pipeline can leverage
+- [train_pipeline.py](train_pipeline.py) This script inherts from base_pipeline.py and is used to construct the pipeline and its steps. The script kickoff_train_pipeline.py will call the function defined here and the pipeline_config
+- [pipeline_config.json](pipeline_config.json) This config specifies the steps of the pipeline, location of data, and any specific arguments. This is consumed once the kickoff_train_script.py is run
+- [train.py](../../../experiments/interpretation/dutchf3_patch/local/train.py) This is the training script that is used to train the model
+- [unet.yaml](../../../experiments/interpretation/dutchf3_patch/local/configs/unet.yaml) This config specifices the model configuration to be used in train.py and is referenced in the pipeline_config.json
+- [azureml_requirements.txt](../../../experiments/interpretation/dutchf3_patch/local/azureml_requirements.txt) This file holds all dependencies for train.py so they can be installed on the compute in Azure ML
+- [logging.config](../../../experiments/interpretation/dutchf3_patch/local/logging.config) This config is used to set up logging
+- local environment with cv_lib and interpretation set up using guidance [here](../../../README.md)
+
+## Running a Pipeline in AzureML
+Set the following environment variables:
+```
+BLOB_ACCOUNT_NAME
+BLOB_CONTAINER_NAME
+BLOB_ACCOUNT_KEY
+BLOB_SUB_ID
+AML_COMPUTE_CLUSTER_NAME
+AML_COMPUTE_CLUSTER_MIN_NODES
+AML_COMPUTE_CLUSTER_MAX_NODES
+AML_COMPUTE_CLUSTER_SKU
+```
+
+On Linux:
+`export VARIABLE=value`
+These can be set automatically in VSCode in an .env file or run with `source .env` in Linux. An example .env file is found at the ROOT of this repo [here](../../../.env.example). You can rename this to .env. Feel free to use this as your .env file but be sure to add this to your .gitignore to ensure you do not commit any secrets. 
```

Comment:
Why is VSCode mentioned? Can we just say "Our code can pick the environment variables from .env file; alternatively you can `source .env` to activate these variables in your environment"

Reply:
It's talking about the environment variable definitions file that can be used in vscode with the python extension: https://code.visualstudio.com/docs/python/environments. We should make that more clear

Reply:
Suggest removing VScode mention entirely, even though I undertand the convenience of using VSCode for this - might confuse end used into thinking that VSCode is needed to run this

Reply:
Modified to remove specific VSCode reference in commit https://github.com/microsoft/seismic-deeplearning/pull/195/commits/eb785a91c176902ac76cb99c0284f91c653f1ffb

Reply:
Looks good